stylesheets/header.scss

```diff
@@ -0,0 +1,189 @@
+.govuk-custom-header {
+  position: relative;
+  z-index: 4;
+  margin-top: 0;
+  padding-bottom: 21px;
+  padding-bottom: 1.3125rem;
+  border-bottom: 2px solid #84bd00;
+  background: #ffffff;
+}
+
+html.govuk-custom-template > body.govuk-custom-template__body * {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.govuk-custom-header__container {
+  display: flex;
+  align-items: center;
+}
+
+.govuk-custom-header__logotype {
+  svg {
+    width: 100%;
+    max-width: 12rem;
+    padding-top: 0.25rem;
+  }
+}
+
+.govuk-custom-table__row--data:hover {
+  cursor: pointer;
+  background-color: rgb(233, 246, 252);
+}
+
+.govuk-custom-header:before {
+  bottom: 3px;
+  border-bottom: 3px solid #00b341;
+}
+
+.govuk-custom-header:after {
+  bottom: 9px;
+  border-bottom: 4px solid #00664f;
+}
+
+.govuk-custom-header:before,
+.govuk-custom-header:after {
+  content: '';
+  display: block;
+  position: absolute;
+  right: 0;
+  left: 0;
+}
+
+.govuk-custom-header__main {
+  color: #ffffff;
+  background: #0b0c0c;
+  padding: 22px 0;
+}
+
+.govuk-custom-container {
+  max-width: 960px;
+  margin: 0 auto;
+}
+
+.govuk-custom-header__wrapper {
+  display: flex;
+  -webkit-box-pack: justify;
+  justify-content: space-between;
+}
+
+.govuk-custom-header__title {
+  display: inline-block;
+  margin-top: 0;
+  margin-right: 24px;
+  margin-right: 1.5rem;
+  margin-bottom: 0;
+}
+
+.govuk-custom-header__title-link {
+  display: flex;
+  margin-top: 0;
+  -webkit-box-align: start;
+  align-items: flex-start;
+  -moz-osx-font-smoothing: grayscale;
+  display: block;
+  padding: 0.25rem 0rem;
+  border-bottom: 1px solid transparent;
+  text-decoration: none;
+}
+
+.govuk-custom-header__links a,
+.govuk-custom-header__links p {
+  margin-top: 0.5rem;
+  margin-bottom: 0;
+  padding: 0.25rem 0rem;
+  color: #ffffff;
+  font-weight: 400;
+}
+
+.govuk-custom-header__links {
+  -webkit-box-pack: end;
+  -webkit-justify-content: flex-end;
+  -ms-flex-pack: end;
+  justify-content: flex-end;
+  display: flex;
+}
+.govuk-custom-header__links a {
+  margin-left: 1.725rem;
+  text-decoration: none;
+}
+
+.govuk-custom-header__service-name {
+  margin-top: 0.5rem;
+  margin-bottom: 0;
+  padding: 0.25rem 0rem;
+  color: #ffffff;
+  font-weight: 400;
+  position: absolute;
+  top: 1.25em;
+  margin-left: 8px;
+}
+
+@media (max-width: 48.0625em) {
+  .govuk-custom-header__links {
+    top: 0.9375em;
+    position: relative;
+  }
+}
+
+@media (min-width: 25.9375em) and (max-width: 48.0625em) {
+  .govuk-custom-header__links a,
+  .govuk-custom-header__links p,
+  .govuk-custom-header__service-name {
+    font-size: 1.1875rem;
+    line-height: 1.875rem;
+  }
+
+  .govuk-custom-header__main {
+    padding: 1.25rem 0rem 2.75rem;
+  }
+
+  .govuk-custom-container {
+    margin-right: max(30px, calc(15px + env(safe-area-inset-right)));
```

Comment:
what `env(safe-area-inset-right)` is doing? never seen this before 🤔

Reply:
Yeah I know. Saw it in the current version